docs/specification/README.md

```diff
@@ -336,23 +338,25 @@ A Tuple<uint16, string>:
 
 ### Array<array_type>
 
-Arrays are serialized using a uint32 byte length followed by the serialized array elements.
+Arrays are serialized using a `uint32` byte length followed by the serialized array elements.
 
     <byte length><serialized element><serialized element>...
 
-An array of uint32 is specified as Array<uint32> and serialized as:
+An array of uint64 is specified as Array<uint64> and serialized as:
+
+    <byte length><uint64><uint64><uint64>...
 
-    <byte length><uint32><uint32><uint32>...
+> Since arrays use a `uint32` byte length prefix, the maximum size of the serialized array elements cannot exceed 4,294,967,295 bytes.
```

Comment:
Either "the maximum size is" or "the size cannot exceed" but not "the maximum size cannot exceed"